css/index.css

```diff
@@ -1,3 +1,257 @@
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+		padding: 22px;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+/* * {
+	box-sizing: border-box;
+} */
+nav {
+	display: flex;
+	justify-content: center;
+	align-items: stretch;
+	justify-content: space-around;
+}
+img.logo {
+	display: inline-block;
+	border: box;
+}
+img.main-image {
+	display: flex;
+	width: 100px;
+	margin: 15px;
+}
+.middle-img {
+	margin: 0 auto;
+}
+a {
+	padding: 40px ;
+	text-decoration: none;
+	font-size: 15px;
+	color: grey;
+}
+h1 {
+	display:flex;
+	font-size: 30px;
+	padding: 20px;
+	margin: 20px;
+}
+div .topLine {
+	display: flex;
+	box-sizing: border-box;
+	align-items: center;
+	justify-content: center;
+	align-content: space-between;
+
+}
+hr {
+	1000px;
+}
+main {
+	margin: 0 auto;
+}
+.lower-content p {
+	margin-right: 10px;
+}
+}
+footer {
+	margin: 20px 0;
+}
+.container {
+	margin: 0 auto;
+	width: 800px;
+	padding-top: 10px;
+}
+
+div .banner {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.main-content section {
+  margin: 20px 1%;
+  width: 45%;
+  display: flex;
+	align-items: center;
+	justify-content: center;
+}
+header .main-image {
+	display: inline;
+	width: 45%;
+}
+div .upper-content{
+	margin-top: 20px;
+	display: flex;
+}
+.lower-content {
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	align-items: center;
+	align-content: space-between;
+	margin-top: 20px;
+	margin-bottom: 20px;
+}
+div.title-section {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	padding: 30px;
+}
+.contact {
+	width: 22%;
+
+}
+.contact address, small{
+	padding: 20px 0;
+	margin:20px 0;
+}
+small {
+	width: 22%;
+}
+.contact .details  {
+	margin: 10px 0;
+	padding: 10px 0;
+}
+@media (max-width: 500px) {
+	.container {
+		width: 100%
+		padding: 0;
+		margin: 0;
+	}
+	.topLine {
+		margin: 0;
+	}
+	body {
+		margin: 0;
+		padding:0;
+	}
+	html {
+		margin: 0;
+		padding:0;
+	}
+	nav {
+		display: flex;
+		flex-direction: column;
+		width: 100%;
+		text-align: center;
+		margin: 0;
+		margin-left: 0;
+		}
+	a {
+		border: 1px black solid;
+		background-color: light-grey;
+		color: black;
+		width: 100%;
+	}
+	.topLine {
+		flex-direction: column;
+	}
+	.topLine img.logo {
+		order: 0;
+	}
+	img.main-image {
+		display: none;
+ }
+ .upper-content {
+	 flex-direction: column;
+ }
+ .lower-content {
+	 flex-direction: column;
+ }
+ .features, .about, .services, .products, .vision {
+	 width: 100%;
+ }
+}
+@media (max-width: 800px){
+	img.main-image {
+		display: none;
+ }
+ body {
+	 width: 100%;
+ }
+ .container {
+	 width:100%;
+
+ }
+ .topLine {
+	 display: flex;
+	 flex-direction: column;
+	 width: 100%;
+	 height: auto;
+ }
+ img {
+	 width: 100%;
+	 height: auto;
+ }
+ nav {
+	 width: 100%;
+	}
+	a {
+		width: 12%;
```

Comment:
If you were to completely take this out, your mobile view will look a little nicer